go.mod

```diff
@@ -23,14 +21,13 @@ require (
 	google.golang.org/appengine v1.6.2 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	istio.io/api v0.0.0-20190820204432-483f2547d882
-	k8s.io/apimachinery v0.0.0-20190923155427-ec87dd743e08
-	k8s.io/client-go v0.0.0-20190923155754-2bd34d193bac
-	k8s.io/code-generator v0.0.0-20190923155300-6206bfaf5c98 // indirect
+	k8s.io/apimachinery v0.0.0-20190612205821-1799e75a0719
```

Comment:
how did you chose these versions?

Reply:
Versions match istio master branch currently: https://github.com/istio/istio/blob/22a34d71694cefcbb6bb310f62eac135eec94914/go.mod#L183